woocommerce/woocommerce-ios
# Dashboard Mark 4: Added site visit stats logic to Networking and Yosemite frameworks

This PR does the following:

* **Networking framework:** Added `SiteVisitStats` mapper, remote, and models
* **Networking framework:** Renamed  `OrderStatGranularity` → `StatGranularity`
* **Networking framework:** Added more unit tests + response JSON _(which is why this PR is so big...sorry!)_

~

* **Yosemite framework:** Refactored `OrderStatsStore` → `StatsStore` and `OrderStatsAction` → `StatsAction`
* **Yosemite framework:** Added site visit-related action logic to `StatsStore` and `StatsAction`
* **Yosemite framework:** `SiteVisitStats` and `SiteVisitStatsItem` are now included in the Yosemite model definition
* **Yosemite framework:** Added more unit tests to support ☝️ 

~

* **WCiOS app:** Added `StatsStore` to `AuthenticatedState`

Ref: #177 

## Testing

* Make sure everything makes sense
* Build and run the app
* Run all of the unit tests and ensure they are ✅ 

@jleandroperez can you take a quick peek at this? Thanks!!!
